blockly-core/core/ui/block_svg/block_svg_functional.js

```diff
@@ -222,6 +222,20 @@ Blockly.BlockSvgFunctional.prototype.renderDrawRightInlineFunctional_ =
   }
 };
 
+Blockly.BlockSvgFunctional.prototype.updateToColour_ = function(hexColour) {
+  goog.base(this, 'updateToColour_', hexColour);
+
+  if (!this.divider_) {
```

Comment:
Not really i dont think. Basically this function says "if we have a divider, do something to said divider". As such, I think checking for the presence of the divider makes the most sense.